configs/clip.yml

```diff
@@ -1,22 +1,35 @@
 # Model
-feature_type: 'clip'
-model_name: 'ViT-B/32' # ViT-B/16 RN50x16 RN50x4 RN101 RN50
+feature_type: "clip"
+model_name: "ViT-B/32" # ViT-B/16 RN50x16 RN50x4 RN101 RN50
 batch_size: 1 # Batchsize (only frame-wise extractors are supported)
 extraction_fps: null # For original video fps, leave unspecified "null" (None)
 extraction_total: null # extract a fix number of frames. It is mutually exclusive with "fps"
 
 # Extraction Parameters
-device: 'cuda:0'  # device as in `torch`, can be 'cpu'
-on_extraction: 'print'  # what to do once the features are extracted. Can be ['print', 'save_numpy', 'save_pickle']
-output_path: './output' # where to store results if saved
-tmp_path: './tmp' # folder to store the temporary files used for extraction (frames or aud files)
+device: "cuda:0" # device as in `torch`, can be 'cpu'
+on_extraction: "print" # what to do once the features are extracted. Can be ['print', 'save_numpy', 'save_pickle']
+output_path: "./output" # where to store results if saved
+tmp_path: "./tmp" # folder to store the temporary files used for extraction (frames or aud files)
 keep_tmp_files: false # to keep temp files after feature extraction.
 show_pred: false # to show preds of a model, i.e. on a pre-train dataset for each feature (Kinetics 400)
 pred_texts: null # provide a list of multiple sentences.  if `null`, will perform zero-shot on Kinetics 400
 
 # config
 config: null
 
+# submitit
```

Comment:
why did you decide to duplicate slurm arguments across all models? could we simply merge the base slurm config to the main config: 
https://github.com/v-iashin/video_features/blob/bd827df5e543ef3270946bcf63e47e18abcdd9ab/main.py#L10

Reply:
Because I didn't know how to do this :sun:
I'll revise this